src/scripts/practice_controller.js

```diff
@@ -0,0 +1,62 @@
+import { Controller } from 'stimulus-repo/packages/stimulus'
+
+export class PracticeController extends Controller {
+  static parent = 'math'
+  static targets = ['answer', 'equation', 'result']
+
+  connect() {
+    this.a = this.randomNumber()
+    this.b = this.randomNumber()
+    this.equationTarget.textContent = `${this.a} * ${this.b}`
```

Comment:
One of the thing that you can start to see more clearly now that these are broken up is which parts of the code is specifically about multiplication. If we want to let it also do division, then those are the parts we'd have to be able to swap out.